lib/class/class_notification_service.dart

```diff
@@ -340,7 +277,7 @@ class NotificationService {
     debugPrint("Notification clicked");
     debugPrint(notificationResponse?.payload);
     onNotification.add(notificationResponse?.payload);
-    dynamic i = 1;
+    dynamic i = 1; //@TODO(Nucleus): What is this for?
```

Comment:
You can just remove this return value. Unsure why it's there at all but right now the value isn't used. `dynamic` is a no-go in general unless there is a _very_ good reason, and in this case the return value of this function (which is invalid, you should never return just `Future` but always `Future<some type>`) isn't used by the caller.